lib/kafka/cluster.rb

```diff
@@ -206,13 +207,14 @@ def fetch_cluster_info
 
           return cluster_info
         rescue Error => e
-          @logger.error "Failed to fetch metadata from #{node}: #{e}"
+          broker_error = "Failed to fetch metadata from #{node}: #{e}"
+          @logger.error broker_error
         ensure
           broker.disconnect unless broker.nil?
         end
       end
 
-      raise ConnectionError, "Could not connect to any of the seed brokers: #{@seed_brokers.join(', ')}"
+      raise ConnectionError, "Could not connect to any of the seed brokers: #{@seed_brokers.join(', ')}: #{broker_error}"
     end
```

Comment:
ah nice! We use ruby-kafka for healthchecking individual brokers, but you're right